components/collapsible/composed/README.md

```diff
@@ -0,0 +1,23 @@
+# CollapsibleComposed
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-collapsible-composed --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import CollapsibleComposed from '@schibstedspain/sui-collapsible-composed'
+
+return (<CollapsibleComposed />)
```

Comment:
Put a real example form playground for instance.
And give a description is possible.

Reply:
done.